samples/client/petstore/typescript-axios/builds/composed-schemas/api.ts

```diff
@@ -0,0 +1,420 @@
+// tslint:disable
+/**
+ * Example
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+import * as globalImportUrl from 'url';
+import { Configuration } from './configuration';
+import globalAxios, { AxiosPromise, AxiosInstance } from 'axios';
+// Some imports not used depending on template conditions
+// @ts-ignore
+import { BASE_PATH, COLLECTION_FORMATS, RequestArgs, BaseAPI, RequiredError } from './base';
+
+/**
+ * 
+ * @export
+ * @interface Cat
+ */
+export interface Cat {
+    /**
+     * 
+     * @type {boolean}
+     * @memberof Cat
+     */
+    hunts?: boolean;
+    /**
+     * 
+     * @type {number}
+     * @memberof Cat
+     */
+    age?: number;
+}
+/**
+ * 
+ * @export
+ * @interface CatAllOf
+ */
+export interface CatAllOf {
```

Comment:
where is this interface used?

Reply:
I think it's not used anywhere, 
I've also noticed that, but testing without the changes in this branch generates the same unused interfaces - so I think it's another bug unrelated to this branch

Reply:
It's probably a manifestation of this old bug: https://github.com/OpenAPITools/openapi-generator/issues/5171